server/src/test/java/com/objectcomputing/checkins/services/feedback_template/FeedbackTemplateControllerTest.java

```diff
@@ -361,6 +361,47 @@ void testGetByTitle() {
         assertContentEqualsEntity(templateTwo, response.getBody().get().get(1));
     }
 
+    @Test
+    void testFindDoesNotGrabReviewsForNonAdmin() {
```

Comment:
This test saves 3 templates, one of which is a review template, then retrieves all templates as a MEMBER and validates that only 2 of the templates are returned (and that it's the right ones).